edi_oca/tests/test_security.py

```diff
@@ -113,7 +113,15 @@ def test_no_group_no_read(self):
     @mute_logger("odoo.addons.base.models.ir_rule")
     def test_rule_no_read(self):
         exchange_record = self.create_record()
-        self.user.write({"groups_id": [(4, self.group.id)]})
+        self.user.write(
+            {
+                "groups_id": [
+                    # To read the related queue jobs
+                    (4, self.env.ref("queue_job.group_queue_job_manager").id),
```

Comment:
This is a sign that the avg user will probably get an error when jumping to the record, no?
Shall we introduce a basic edi user group and imply the read access on jobs for it?
As an alternative I would protect the field and the action with the manager group.